nominal/core/_clientsbunch.py

```diff
@@ -64,3 +69,39 @@ def from_config(cls, cfg: ServiceConfiguration, agent: str, token: str) -> Self:
             upload=client_factory(upload_api.UploadService),
             video=client_factory(scout_video.VideoService),
         )
+
+    def auth_expiration(self) -> datetime.datetime:
```

Comment:
I don't think I want this exposed as a primary method, because I don't believe all tokens that people will handle will be jwts. For automated scripts, we'll likely be moving towards _API keys_ instead which I'm not clear yet if they will be jwts